.github/workflows/e2e_additional_editor_and_player.yml

```diff
@@ -38,9 +38,13 @@ jobs:
       - name: Run Additional Editor E2E Test
         if: ${{ steps.check-risk.outputs.is-low-risk != 0 }}
         run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_e2e_tests --suite="additionalEditorFeatures" --prod_env
+        env: 
+          OPPIA_VIDEOS: 0
```

Comment:
Can we use a boolean here (see [this](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#literals))?

Also, can we rename this accordingly (e.g. `VIDEO_RECORDING_IS_ENABLED`)?

Reply:
@kevintab95 

Apparently the package I'm using to read the environment variables (`dotenv`) saves all environment variables as strings. See [this](https://github.com/motdotla/dotenv/issues/51)

This is why the PR isn't working as expected (see any of the below runs, e2e videos are still being created).

Should I go back to setting the value of the variable to `0` and `1` instead (it worked then)?

Reply:
Ah, I see. In that case, I'm fine with using  a 0/1 boolean but it would be nice to indicate it in the key (and maybe add this info / issue in a comment for developers). Thanks for looking into this!